spec/javascripts/health-care/health-care.spec.js

```diff
@@ -201,5 +211,31 @@ describe("Health Care Form", function() {
         expect(true).to.be.false;
       }).always(function() { done(); });
   });
+
+  it("should return true for valid monetary value input", function() {
+    var HealthApp = document.getElementById("fixture-frame").contentWindow.HealthApp;
+    var form = HealthApp.getFormRoot();
+    var testElement = form.elements['veteran[gross_wage_income]'];
+    var validInput = ['55000.00', '100', '1.99'];
+
+    for (var i = 0; i < validInput.length; i++) {
+      testElement.value = validInput[i];
+      var valid = HealthApp.validate(testElement);
+      expect(valid).to.be.true;
+    }
+  });
+
+  it("should return false for invalid monetary value input", function() {
+    var HealthApp = document.getElementById("fixture-frame").contentWindow.HealthApp;
+    var form = HealthApp.getFormRoot();
+    var testElement = form.elements['veteran[gross_wage_income]'];
+    var invalidInput = ['1,000', 'abc', '$100'];
+
+    for (var i = 0; i < invalidInput.length; i++) {
+      testElement.value = invalidInput[i];
+      var valid = HealthApp.validate(testElement);
+      expect(valid).to.be.false;
+    }
+  });
```

Comment:
I'm torn on whether or not we want a unittest that ensure the validation is on each dom node we care about. The test would basically be listing each DOM node and checking of the data attribute was set to monetary. Overkill or not? Defer to your judgement.